woof-code/rootfs-skeleton/usr/bin/new2dir

```diff
@@ -235,27 +260,66 @@ if [ "$auto" = 0 ];then
 	echo "(and optionally ${DEV_TARGETDIR}, ${DOC_TARGETDIR}, ${NLS_TARGETDIR})"
 	echo -n "Press ENTER key to continue: "
 	read goforit
-	
+
 	installwatch -o /tmp/pkginstall.list ${@}
 else
 	installwatch -o /tmp/pkginstall.list ${@}
 fi
+
 sync
 #create list of installed files...
-cat /tmp/pkginstall.list | grep '#success$' | tr -s '\t' | tr '&' ' ' | tr '\t' '&' | egrep '^[345]&open&|^0&chmod&' | grep --extended-regexp -v '&/dev/tty&|&/dev/null&|&/root/\.packages/|&/tmp/|&/root/\.icewm/|&/proc/|&/sys/|DotPupTmpDir|/\.myownmenuerc' | grep -E -v '&/initrd|&/mnt/' | cut -f 3 -d '&' | sort -u > ${RELPATH}/${EXE_PKGNAME}.files #120220 120602
+cat /tmp/pkginstall.list \
+  | grep '#success$' \
+  | tr -s '\t' \
+  | tr '&' ' ' \
+  | tr '\t' '&' \
+  | egrep '^[345]&open&|^0&chmod&' \
+  | grep --extended-regexp -v '&/dev/tty&|&/dev/null&|&/root/\.packages/|&'"$PWD"'|&/root/'"$PWD"'|&/tmp/|&/root/\.icewm/|&/proc/|&/sys/|DotPupTmpDir|/\.myownmenuerc' \
```

Comment:
The `|&'"$PWD"'|&/root/'"$PWD"'` in _each_ of the grep regexes (more below) may be overly cautious here, but it worked for me to help strip out build time stuff that sometimes ends up in the PET itself..